openshift.py

```diff
@@ -0,0 +1,750 @@
+import kubernetes
+import pprint
+import logging
+import requests
+import json
+import re
+from flask import Flask, redirect, url_for, request, Response
+
+import sys
+
+# application = Flask(__name__)
+
+
+class openshift:
+    headers = None
+    verify = False
+    url = None
+
+    def __init__(self, url, token, logger):
+        self.set_token(token)
+        self.set_url(url)
+        self.logger = logger
+
+    def set_token(self, token):
+        self.headers = {
+            "Authorization": "Bearer " + token,
+            "Accept": "application/json",
+            "Content-Type": "application/json",
+        }
+
+    def set_url(self, url):
```

Comment:
Opportunity to be more Pythonic: If you want these to behave like java setters and getters, you can make it a python property. Otherwise, just directly accessing the url attribute is fine.